scripts/build/toolchain/termux_setup_toolchain_23c.sh

```diff
@@ -181,6 +181,10 @@ termux_setup_toolchain_23c() {
 		fi
 	done
 
+	if [ "$TERMUX_CACHE_ENABLED" = "true" ]; then
+		sed -i 's|"$bin_dir/clang"|ccache "$bin_dir/clang"|g' \
+			$_TERMUX_TOOLCHAIN_TMPDIR/bin/*-{clang,clang++,cpp,gcc,g++}
+	fi
```

Comment:
Seems like caching won't work locally if user doesn't set TERMUX_CACHE_ENABLED when setting up the toolchain with this. I think we should add a patch to move the if into the `*-{clang,clang++,cpp,gcc,g++}` scripts instead, so that we end up with something like (not yet tested):

```sh
$ cat ~/.termux-build/_cache/android-r25c-api-24-v2/bin/aarch64-linux-android24-clang
#!/bin/bash
bin_dir=`dirname "$0"`
if [ "$TERMUX_CACHE_ENABLED" = "true" ]; then
    clang="ccache $bin_dir/clang"
else
    clang="$bin_dir/clang"
fi

if [ "$1" != "-cc1" ]; then
    "$clang" --target=aarch64-linux-android24 "$@"
else
    # Target is already an argument.
    "$clang" "$@"
fi
```